src/main/java/domain/PurchasedLotto.java

```diff
@@ -0,0 +1,57 @@
+package domain;
+
+import domain.strategy.PurchaseStrategy;
+import domain.strategy.RandomPurchaseStrategy;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PurchasedLotto {
+
+    private static final int LOTTO_PRICE = 1000;
+
+    private List<Lotto> lottos = new ArrayList<>();
+    private PrizeResult prizeResult = new PrizeResult();
+    private int inputMoney;
+
+    public PurchasedLotto(int inputMoney) {
+        this.inputMoney = inputMoney;
+        purchaseMaximunLottos();
+    }
+
+    private void purchaseMaximunLottos() {
+        for (int i = 0; i < inputMoney / LOTTO_PRICE; i++) {
+            purchase(new RandomPurchaseStrategy());
+        }
+    }
+
+    public void purchase(PurchaseStrategy purchaseStrategy) {
+        Lotto lotto = new Lotto(purchaseStrategy.getNumbers());
+        lottos.add(lotto);
+    }
+
+    public PrizeResult calculatePrizeResult(WinningNumber winningNumber) {
+        lottos.stream()
```

Comment:
```suggestion
        lottos
```
`foreach`를 쓰신다면 `stream`은 삭제해도 무방해요